packages/lesswrong/server/vulcan-lib/apollo-server/context.ts

```diff
@@ -29,7 +30,7 @@ const getUser = async loginToken => {
 
     const hashedToken = Accounts._hashLoginToken(loginToken)
 
-    const user = await Meteor.users.rawCollection().findOne({
+    const user = Users.findOne({
```

Comment:
I feel like the reason this must have been here is because this function must be called at some point before the Vulcan collections are initialized. How thoroughly did you test that this isn't breaking everything horribly?

Reply:
This line is weird in this way because it began its life in apollographql's `meteor-integration` package, which would not have had a `Users` object available to import.

Reply:
Ah, cool. That resolves it. Approved.